contentcuration/contentcuration/migrations/0001_initial.py

```diff
@@ -92,7 +92,7 @@ class Migration(migrations.Migration):
                 'verbose_name_plural': 'Topics',
             },
             managers=[
-                ('_default_manager', django.db.models.manager.Manager()),
+                ('objects', django.db.models.manager.Manager()),
```

Comment:
This was required for upgrading django-mptt... see [here](http://django-mptt.github.io/django-mptt/upgrade.html)

Reply:
Would this ruin already migrated databases?

Reply:
Honestly, I don't fully understand what's going on with this change.  I just know that after upgrading, I was prompted to run migrations.  I tried running them and got an error.  I googled the error and found that upgrade note in the django-mptt docs.  After making the change, I was able to run the migrations smoothly.

So I did actually

Reply:
After thinking about this more, i think 1.11 just doesn't have the `_default_manager` attribute, so this really shouldn't affect the actual migrations at least.